sigridci/sigridci/sigrid_api_client.py

```diff
@@ -15,6 +15,7 @@
 import json
 import os
 import ssl
+import urllib.error
```

Comment:
Why this import while nothing else changed in this file (except on line 171, but that seems irrelevant).